extensions/amp-iframe/0.1/test/test-amp-iframe.js

```diff
@@ -402,7 +402,7 @@ describes.realWin(
         });
       });
 
-      describe('allow-same-origin w/o same-origin-iframe', () => {
+      describe('allow-same-origin', () => {
         // TODO(#30824): remove when same-origin-url is launched.
```

Comment:
Does this TODO need to be removed / updated?

Reply:
Yep. That's right. Thanks for catching.